doc/releases/changelog-dev.md

```diff
@@ -17,6 +17,13 @@
 * Alias for Identity (`I`) is now accessible from `qml.ops`.
   [(#7200)](https://github.com/PennyLaneAI/pennylane/pull/7200)
 
+<h3>Labs: a place for unified and rapid prototyping of research software 🧪</h3>
+
+* A `parity_matrix` function is now available in `pennylane.labs.phase_polynomials`.
```

Comment:
Have there been discussions about the submodule name? Maybe something like `intermed_reps` or so is more versatile for extensions/reusage in the future?

Reply:
I like the idea 👌

Reply:
although right now I dont think there are any plans for more IRs right?

Reply:
Yeah, I also noticed that there are only 2 planned and the parity matrix can be considered a part of phase polynomials. However, as the two functionalities/IRs are separate, and the phase polynomial one does not explicitly use the parity matrix one, I personally would still find it more intuitive to have the name be more general.

Reply:
yeah good point, I renamed it to `intermediate_reps` now